src/Literate.jl

```diff
@@ -504,6 +507,44 @@ function execute_markdown!(io::IO, sb::Module, block::String, outputdir)
     end
 end
 
+function execute_markdown!(io::IO, flavor::Franklin, sb::Module, block::String, outputdir)
+    # TODO: Deal with explicit display(...) calls
+    r, str, _ = execute_block(sb, block)
+    plain_fence = "\n```\n" =>  "\n```" # issue #101: consecutive codefenced blocks need newline
+    if r !== nothing && !REPL.ends_with_semicolon(block)
+        for (mime, ext) in [(MIME("image/svg+xml"), ".svg"), (MIME("image/png"), ".png"), (MIME("image/jpeg"), ".jpeg")]
```

Comment:
@jkrumbiegel is this fine for svg output or is it better to use something like https://github.com/JuliaDocs/Documenter.jl/pull/1538? I only included `text/html` in my PR for now.